packages/flutter/lib/src/gestures/recognizer.dart

```diff
@@ -632,9 +639,21 @@ abstract class PrimaryPointerGestureRecognizer extends OneSequenceGestureRecogni
   /// and signaling subsequent callbacks.
   ///
   /// Can be null to indicate that the gesture can drift for any distance.
-  /// Defaults to 18 logical pixels.
+  /// Defaults to touch slop from [gestureSettings] (18 logical pixels).
```

Comment:
This should be 
```suggestion
  /// Defaults to touch slop from [kTouchSlop] (18 logical pixels).
```
because gestureSettings is null unless provided.

Reply:
I reverted my edit as you suggested, but I think it would be more helpful to describe that when the parameter is unspecified, it defaults to `gestureSettings` and falls back to `kTouchSlop`.